keps/sig-scheduling/5471-enable-sla-based-scheduling/README.md

```diff
@@ -0,0 +1,855 @@
+      # Only accept GPUs with SLA >= 950 (95%)
+      - key: node.kubernetes.io/sla
+        operator: Gt
+        value: "950"
+        effect: NoSchedule
+---
+# Pod using DRA claim with SLA requirements
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dra-workload
+spec:
+  resourceClaims:
+  - name: gpu-claim
+    resourceClaimName: gpu-claim-high-sla
+  tolerations:
+  - key: node.kubernetes.io/sla
+    operator: Gt
+    value: "950"  # Ensure GPU nodes meet SLA requirements
+    effect: NoSchedule
+  containers:
+  - name: ml-workload
+    resources:
+      claims:
+      - name: gpu-claim
+```
+
+#### Story 5 — DRA device-level error budget management
+
+As a platform engineer managing GPU clusters with varying reliability states, I want to allocate devices based on their remaining error budget using numeric tolerations. So that critical workloads only get devices with sufficient reliability headroom while allowing degraded devices to serve less sensitive workloads.
+
+This will get the critical inference fresh devices (>24h error budget), batch training can use aging devices (1-24h), and severely degraded devices (<1h) are excluded from allocation entirely, enabling graceful device lifecycle management.
+
+**Example Configuration:**
+
+```yaml
+# Driver taints devices with low error budget
+kind: ResourceSlice
+spec:
+  driver: device.example.com
+  devices:
+  - name: gpu-node-01-device-0
+    attributes:
+      memory: "32Gi"
+      compute-capability: "8.6"
+    # Driver applies taint when error budget drops below 10 hours
+    taints:
+    - key: device.example.com/error-budget-in-hours
+      value: "8"  # 8 hours remaining
+      effect: NoSchedule
+---
+# Critical inference workload requires high-reliability devices
+kind: ResourceClaim
+metadata:
+  name: inference-gpu-claim
+spec:
+  requests:
+  - name: high-reliability-gpu
+    deviceClassName: device.example.com
+    tolerations:
+    # Only accept devices with >24 hours error budget
+    - key: device.example.com/error-budget-in-hours
+      operator: Gt
+      value: "24"
+      effect: NoSchedule
+---
+# Batch Short-lived batch training workload tolerates degraded devices
+kind: ResourceClaim
+metadata:
+  name: training-gpu-claim
+spec:
+  requests:
+  - name: batch-gpu
+    deviceClassName: device.example.com
+    tolerations:
+    # Accept devices with >1 hour error budget
+    - key: device.example.com/error-budget-in-hours
+      operator: Gt
+      value: "1"
+      effect: NoSchedule
+```
+
+### Notes/Constraints/Caveats (Optional)
+
+- **Integer-Only Support**: The implementation supports signed 64-bit integers only. Pod specs containing toleration values with decimal numbers (e.g., `"95.5"`) will be rejected by the API server during validation when using numeric comparison operators.
+
+- **Parsing Requirements**: The toleration value must be parseable as integers for numeric operators (`Lt`, `Gt`). If fails parsing, the toleration does not match.
```

Comment:
These are saying the same thing? As the first point said, we'll do a validation when a toleration uses Lt or Gt. So, the parse should never fail (unless we have bugs), right?